charts/k0s/values.yaml

```diff
@@ -223,4 +223,23 @@ securityContext:
 # to manage service endpoints
 openshift:
   enable: false
-  
\ No newline at end of file
+
+isolation:
+  enabled: false
+
+  resourceQuota:
+    quota:
+      cpu: 10
+      memory: 20Gi
+      pods: 10
+    scopeSelector:
+      matchExpressions:
+    scopes:
+  
+  limits:
+    default:
```

Comment:
please add an option `isolation.limitRange.enabled: true` so that people could disable the limit range if wanted